src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -83,7 +83,6 @@ class ScenarioTest(ReplayableTest, CheckerMixin, unittest.TestCase):
     def __init__(self, method_name, config_file=None, recording_name=None,
                  recording_processors=None, replay_processors=None, recording_patches=None, replay_patches=None,
                  random_config_dir=False):
-        self.cli_ctx = get_dummy_cli(random_config_dir=random_config_dir)
```

Comment:
Some tests reported the following errors:
`AttributeError: 'xxxScenarioTest' object has no attribute 'cli_ctx'`

Reply:
`cli_ctx` is moved to `setUp` mothod, which is executed after `__init__`.
If `cli_ctx` is used in `__init__`, this error is raised.